### DISCUSSION TITLE
v.2.2.0 - 25 new languages added

### DISCUSSION THREAD

**Discussion by Blackymas** [Blackymas/NSPanel_HA_Blueprint#133] [Announcements]
🎉🎊🎉 HAPPY NEW YEAR 🎉🎊🎉

## General
I am speechless and proud of how the project has developed. I never expected so many people to be interested in my project.

A big thank you goes out here once again to @deejaybeam and [SmartHome yourself](https://www.youtube.com/watch?v=3afPFg6kUdc) aka Daniel. ❤️❤️❤️

Appropriate to the end of the year we have implemented two things that were very popular. We have added some new languages and there is now a setup video in German (I hope we will find in the next weeks also an English speaking YouTuber).

&nbsp;

## Files that need to be reloaded:

- nspanel_blueprint.yaml
- nspanel.tft

&nbsp;

## What's changed

- [German Setup Video](https://www.youtube.com/watch?v=3afPFg6kUdc) 
- [Github Discussion and Announcements ](https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions)
- add 25 new languages
'English'
'Germany'
'Bulgaria'
'Denmark'
'Estonia'
'Finland'
'France'
'Greece'
'Indonesia'
'Italy'
'Latvia'
'Lithuania'
'Netherlands'
'Poland'
'Portugal'
'Romania'
'Sweden'
'Slovakia'
'Slovenia'
'Spain'
'Czechia'
'Turkey'
'Ukraine'
'Hungary'
'Norway'

&nbsp;

## Next topics we are currently working on

- Notifications + Notifications Page
- New Home Page for more features

<hr /><em>This discussion was created from the release <a href='https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v.2.2.0'>v.2.2.0 - 25 25 new languages added</a>.</em>

**Comment by nijez** [comment]
I see that Chinese is not supported yet. I want to provide Chinese support, what should I do?

**Reply by kuriburi** [reply]
Same for me, I would like to contribute Japanese to the list of supported languages.

**Reply by edwardtfn** [reply]
[Chinese, Japanese and Korean (CJK) languages - We need help with translations · Blackymas/NSPanel_HA_Blueprint · Discussion #1447 (github.com)](https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/1447)
